.github/workflows/node-flow-pull-request-checks.yaml

```diff
@@ -90,6 +90,7 @@ jobs:
 
   eet-tests:
     name: E2E Tests
+    if: ${{ false }}
```

Comment:
These changes not mentioned in PR comment

Reply:
Added a comment. E2E Tests are not relevant for modular code.